src/acquisition/covidcast/database.py

```diff
@@ -337,12 +338,18 @@ def delete_batch(self, cc_deletions):
         raise Exception(f"Bad deletions argument: need a filename or a list of tuples; got a {type(cc_deletions)}")
       self._cursor.execute(add_id_sql)
       self._cursor.execute(delete_sql)
+      total = self._cursor.rowcount
       self._cursor.execute(fix_latest_issue_sql)
       self._connection.commit()
+
+      if total is None or total == -1:
```

Comment:
Isn't this `is None` condition redundant?

Reply:
No; we want this block to run only when total is -1 or when total is None, and not when total is 0. 0 is a falsy value in Python.

Reply:
Hah! Good catch. This was pattern-replicating from
https://github.com/cmu-delphi/delphi-epidata/blob/442b3b9ff476c3a1d0c5fe13d16d91458a98624c/src/acquisition/covidcast/database.py#L236-L238

where `is None` is necessary to avoid the else clause... but we don't have one of those here, so it's purely vestigial.

```suggestion
      if total == -1:
```